src/main/scala/io/iohk/ethereum/jsonrpc/server/http/InsecureJsonRpcHttpServer.scala

```diff
@@ -2,23 +2,34 @@ package io.iohk.ethereum.jsonrpc.server.http
 
 import akka.actor.ActorSystem
 import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.RemoteAddress
 import ch.megard.akka.http.cors.scaladsl.model.HttpOriginMatcher
+import com.twitter.util.LruMap
 import io.iohk.ethereum.jsonrpc._
 import io.iohk.ethereum.jsonrpc.server.controllers.JsonRpcBaseController
 import io.iohk.ethereum.jsonrpc.server.http.JsonRpcHttpServer.JsonRpcHttpServerConfig
 import io.iohk.ethereum.utils.Logger
 
 import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.FiniteDuration
 import scala.util.{Failure, Success}
 
-class BasicJsonRpcHttpServer(
+class InsecureJsonRpcHttpServer(
     val jsonRpcController: JsonRpcBaseController,
     val jsonRpcHealthChecker: JsonRpcHealthChecker,
     config: JsonRpcHttpServerConfig
 )(implicit val actorSystem: ActorSystem)
     extends JsonRpcHttpServer
     with Logger {
 
+  override val ipTrackingEnabled: Boolean = config.ipTrackingEnabled
```

Comment:
With the proposal of JsonRpcHttpServer, we need to add the next val, and remove the overridden values:

val config: JsonRpcHttpServerConfig